yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -318,7 +318,7 @@ def _ortho_pixelize(self, data_source, field, bounds, size, antialias,
                             chunk[ptype, px_name].in_units('cm'),
                             chunk[ptype, py_name].in_units('cm'),
                             chunk[ptype, 'smoothing_length'].in_units('cm'),
-                            chunk[ptype, 'particle_mass'].in_units('g'),
+                            chunk[ptype, 'mass'].in_units('g'),
```

Comment:
I'm a little bit worried about this. Projection in yt-4.0 has been working for me. Although when looking at this I started to wonder why. I'll need to do some experiments to figure that out.

Reply:
`mass` is just an alias to `particle_mass`, so there shouldn't be any issue here, correct?

Reply:
I noticed that for SPH particle type like `'gas'`, there is only `'mass'` but no `'particle_mass'` field. Similarly, all the `'particle_xxx'` fields are renamed without the `'particle_'` prefix for the SPH particle type. This must be a decision made earlier in the yt-4.0 refactoring.